src/grandorgue/dialogs/GOMidiListDialog.cpp

```diff
@@ -29,13 +28,12 @@ GOMidiListDialog::GOMidiListDialog(
   GODocumentBase *doc,
   wxWindow *parent,
   const std::vector<GOMidiConfigurator *> &midi_elements)
-  : wxDialog(
+  : GOSimpleDialog(
     parent,
```

Comment:
```suggestion
    parent, wxT("MIDI Objects"), _("MIDI Objects"), 0, wxOK | wxHELP),
```

Othewwist the pre-commit check failes: https://github.com/GrandOrgue/grandorgue/actions/runs/5495417485/jobs/10022178525?pr=1587